### DISCUSSION TITLE
Snowball tar extraction as presigned-url

### DISCUSSION THREAD

**Discussion by Dubkoslove** [minio/minio#19431] [Q&A]
Hello,

I would like to ask you, whether it is possible to create pre-signed URL for uploading/downloading multiple-files in .tar file.

**Use cases:**
1. I want to upload around 500 smaller files (one folder) in one request.
2. I want to download multiple files (e.g. filtered by name)

The solution should be with using pre-signed URLs, is that possible somehow ? If not, what would be suggested solution for this ?  

Thanks.

**Comment by klauspost** [comment]
A) I don't see why that wouldn't be possible.
B) I haven't tried it.
C) Do note that you are giving a wildcard to write anywhere in the bucket unless you are setting `X-Amz-Meta-Minio-Snowball-Ignore-Dirs: true` and `X-Amz-Meta-Minio-Snowball-Prefix: true`.
